.github/workflows/close-old-issues.yml

```diff
@@ -1,12 +1,16 @@
 name: Close Old Issues
 on:
+  schedule:
+    - cron: "0 1 * * *"
```

Comment:
this is 6pm PDT, I think it would be good to run it overnight in PDT, e.g.  8am UTC

Reply:
good suggestion, I'll fix that

Reply:
this is fixed!